### PR TITLE
feat: Swl 14, file upload on http/2 

### PR DESCRIPTION
**Pull request by adeebbayat**
## Checklist

- [ ] I've followed the [Contributing guidelines](https://github.com/community-health-demo/.github/blob/main/CONTRIBUTING.md)
- [ ] I've titled my PR according to the [Conventional Commits spec](https://conventionalcommits.org)
- [ ] I've linked an open issue
- [ ] I've added tests that fail without this PR but pass with it
- [ ] I've linted and tested my code
- [ ] I've updated documentation (if appropriate)

### CODE REVIEW

File: test/testSuite.js [hunk 1]
```diff
@@ -62,12 +62,14 @@ describe('Electron UI Rendering', function () {
 describe('Protocol selection and usage', function () {
   //group of tests that focuses on how application handles different protocols
   reqInputTests();
+
```
Comment (MUnderbrink90) [inline test/testSuite.js:65]: Remove line

File: test/testSuite.js [hunk 2]
```diff
@@ -62,12 +62,14 @@ describe('Electron UI Rendering', function () {
...
   //group of tests that focuses on how application handles different protocols
   reqInputTests();
+
   httpTest();
   graphqlTest();
   websocketTest();
   grpcTest();
   webRTCTest();
   openAPITest();
+
```
Comment (MUnderbrink90) [inline test/testSuite.js:72]: remove line

File: test/__tests__/httpTest.js [hunk 3]
```diff
@@ -166,4 +167,8 @@ describe('REST API Requests', () => {
       expect(ReqResCtrl.toggleSelectAll()).not.toThrowError;
     });
   });
+
```
Comment (MUnderbrink90) [inline test/__tests__/httpTest.js:170]: remove lines 170-173

File: test/__tests__/httpTest.js [hunk 4]
```diff
@@ -1,5 +1,6 @@
 import ReqResCtrl from '../../src/client/controllers/reqResController';
-
+// import { render, fireEvent } from '@testing-library/react';
```
Comment (MUnderbrink90) [inline test/__tests__/httpTest.js:2]: remove commented out lines on 2-3

**Review by MUnderbrink90 — COMMENTED** [review]
Please make corrections to remove commented or blank lines and resubmit for PR

File: test/testSuite.js (hunk 1 above)
Comment (adeebbayat) [reply test/testSuite.js:65]: Done

File: test/testSuite.js (hunk 2 above)
Comment (adeebbayat) [reply test/testSuite.js:72]: Done

**Pull request closed without merging**